opm/parser/eclipse/EclipseState/IOConfig/RestartConfig.cpp

```diff
@@ -195,6 +205,118 @@ namespace Opm {
     }
 
 
+    void RestartConfig::update( size_t step, const RestartSchedule& rs) {
+        if (step == 0)
+            this->restart_schedule->updateInitial( rs );
+        else
+            this->restart_schedule->update( step, rs );
+    }
+
+
+    const std::vector<std::string> RestartConfig::integerKeywords = {""     ,      //  1 BASIC=?? - ignored
```

Comment:
> Why is this a member in the first place?

Old habits.

> static constexpr const char* integerKeywords[] = { ... }. (or anon namespace as you see fit).

I'll update - guess I'll go for the anon namespace.

This patch has some miles to go anyway.

As a side comment: I would encourage anyone skeptical to the value of tests to refactor this code; I have been guided semicolon by semicolon by the tests!